apps/wing/vitest.config.ts

```diff
@@ -4,6 +4,7 @@ export default defineConfig({
   test: {
     // Some tests use process.chdir which is not supported in vitest's multi-thread mode
     threads: false,
+    update: true,
```

Comment:
Should we be consistent about this for all of the packages in the repo?

Reply:
Whoops, should not have left that in. The consistent thing to do would be to update the package.json script, so I'll do that